docs/getting-started.md

```diff
@@ -27,75 +27,336 @@ npm create @workleap/project@latest <output-directory>
 
 1. A module should always match a subdomain of the application business domain and should only export pages.
+
 2. A module should be fully autonomous. It shouldn't have to coordinate with other parts of the application for things as trivial as navigation links.
+
 3. A federated application should feel homogenous. Different parts of a federation application should have the ability to communicate with each others and react to changes happening outside of their boundaries.
+
 4. Data and state should never be shared between parts of a federated application. Even if two parts needs the same data or the same state values, they should load, store and manage those independently.
 
 ## Quick start
 
+### Prerequisites
+
+Before going further make sure you:
+
+- Installed at least the latest LTS version of [Node.js](https://nodejs.org/)
+- Created a project for an host application (we'll call ours `host` in this tutorial)
+- Created a project for a remote module application (we'll call ours `remote-module` in this tutorial)
+
 ### Install the packages
 
-Open a terminal at the root of the host application project and install the following packages:
+Open a terminal at the root of the **host application** project and install the following packages:
 
 +++ pnpm
 ```bash
-pnpm add @squide/core @squide/react-router @squide/webpack-module-federation
+pnpm add @squide/core @squide/react-router @squide/webpack-module-federation webpack react-router-dom
 ```
 +++ yarn
 ```bash
-yarn add @squide/core @squide/react-router @squide/webpack-module-federation
+yarn add @squide/core @squide/react-router @squide/webpack-module-federation webpack react-router-dom
 ```
 +++ npm
 ```bash
-npm install @squide/core @squide/react-router @squide/webpack-module-federation
+npm install @squide/core @squide/react-router @squide/webpack-module-federation webpack react-router-dom
 ```
 +++
 
-Then open a terminal at the root of every remote module project and install the following packages:
+Then open a terminal at the root of every **remote module** project and install the following packages:
 
 +++ pnpm
 ```bash
-pnpm add @squide/core @squide/react-router @squide/webpack-module-federation
+pnpm add @squide/core @squide/react-router @squide/webpack-module-federation webpack react-router-dom
 ```
 +++ yarn
 ```bash
-yarn add @squide/core @squide/react-router @squide/webpack-module-federation
+yarn add @squide/core @squide/react-router @squide/webpack-module-federation webpack react-router-dom
 ```
 +++ npm
 ```bash
-npm install @squide/core @squide/react-router @squide/webpack-module-federation
+npm install @squide/core @squide/react-router @squide/webpack-module-federation webpack react-router-dom
 ```
 +++
 
 ### Configure the host application
```

Comment:
Not sure why there is no Configure a static module also.

I guess if it's because it's not clear what the difference between a static module and a remote module is, we could add it in a Overview kind of section like next.js

Reply:
> Do you think we could split this .md in 3 files? 350+ lines is a lot.
> 
> I could see a Getting Started Section with 3 sections in it:
> 
> * Installation
> * Configure the host application
> * Configure a remote application
> 
> This way each file has a previous/next link, and it help better scope the page
> 
> I'm looking at documentation like vercel and react query ![image](https://user-images.githubusercontent.com/38871812/242892768-0a2f7a8e-4408-48ff-ae18-5e10a1f8f598.png) ![image](https://user-images.githubusercontent.com/38871812/242893210-c2e06234-3230-42e1-b214-3c2eb1b194ef.png)

I'm not sure about this, personally I prefer to have my whole getting started in the same page. It's not that much content, the issue is that the code sample are long.

The "getting started" is also the short version of the getting started, I will add a guide with the longuer version.

Regarding the local module, it will be added as a guide as it's not the most common path so it shouldn't be in the "getting-started" tutorial.

Reply:
> Not sure why there is no Configure a static module also.
> 
> I guess if it's because it's not clear what the difference between a static module and a remote module is, we could add it in a Overview kind of section like next.js

It will be added as a guide as it's not the most common path so it shouldn't be in the "getting-started" tutorial.

Reply:
Ok, I'll try something for the getting-started

Reply:
But one of the issue between for example the Next.js docs and the squide docs, is that they have a lot more to say per section which make sense to have a lot of pages.

For squide, some pages would be so small that it would almost look funny to split them.

Reply:
Done, it is indeed better!